orttraining/orttraining/models/runner/pipeline.cc

```diff
@@ -18,335 +20,667 @@ namespace onnxruntime {
 namespace training {
 namespace pipeline {
 
-Slot::Slot() {
-  type = Empty;
-  batch_id = 0;
+bool Action::IsForward() const {
```

Comment:
For small function, maybe we can try use inline?

Reply:
Good idea.

---
In reply to: [455407917](https://github.com/microsoft/onnxruntime/pull/4524#discussion_r455407917) [](ancestors = 455407917)